From 5fa99b5271a3c80a13c78062d7adc6310222dd8e Mon Sep 17 00:00:00 2001
From: Daniel Bermond <dbermond@archlinux.org>
Date: Mon, 14 Jul 2025 15:20:14 -0300
Subject: [PATCH] Add missing header for GCC 15

The header '<cstdint>'[1] is explicitly required to be included
with GCC 15 and beyond when using fixed width integer types[2].

For more details, see the guide about porting[3] to GCC 15.

[1] https://en.cppreference.com/w/cpp/header/cstdint.html
[2] https://en.cppreference.com/w/cpp/types/integer.html
[3] https://gcc.gnu.org/gcc-15/porting_to.html#cxx
---
 lib/include/ultrahdr/ultrahdrcommon.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/include/ultrahdr/ultrahdrcommon.h b/lib/include/ultrahdr/ultrahdrcommon.h
index 48238441..ee20e773 100644
--- a/lib/include/ultrahdr/ultrahdrcommon.h
+++ b/lib/include/ultrahdr/ultrahdrcommon.h
@@ -24,6 +24,7 @@
 #include <GLES3/gl3.h>
 #endif
 
+#include <cstdint>
 #include <deque>
 #include <map>
 #include <memory>
